llvm_cm/test/X86/bb-frequency.s

```diff
@@ -1,5 +1,4 @@
 ## LLVM-CM frequency weighting test.
-# REQUIRES: x86
```

Comment:
why

Reply:
Was testing things and removed it, figuring that this isn't likely to get run on anything other than X86 currently. I've switched it to use `x86-registered-target` as I believe that's all it should need. (Unless the tflite models don't work on something like AArch64?).

Reply:
They would on that, as in afaik the interpreter can be built there, but probably won't on other hosts for which llvm can be compiled

Reply:
Ack. Switched to only running on `x86-64` using that native lit requirements.